src/main/java/com/stuypulse/robot/constants/Settings.java

```diff
@@ -243,9 +246,9 @@ public interface Wrist {
 
             SmartNumber SHOULDER_VELOCITY_FEEDBACK_CUTOFF = new SmartNumber("Arm/Wrist/Shoulder Velocity Feedback Cutoff (deg per s)",10.0);
 
-            SmartNumber SHOULDER_VELOCITY_FEEDBACK_DEBOUNCE = new SmartNumber("Arm/Wrist/Feedback Enabled Debounce", 0.0);
+            SmartNumber SHOULDER_VELOCITY_FEEDBACK_DEBOUNCE = new SmartNumber("Arm/Wrist/Feedback Enabled Debounce", 0.15);
```

Comment:
Make sure this isn't different for other autons

Reply:
This is in main, need to pull from main